### DISCUSSION TITLE
1.0.6

### DISCUSSION THREAD

**Discussion by boardthatpowder** [aws/aws-connected-device-framework#54] [Announcements]
```
@cdf/assetlibrary 				6.0.1
@cdf/assetlibrary-client			5.0.0
@cdf/assetlibrary-export			3.0.0
@cdf/assetlibrary-history			5.0.0
@cdf/assetlibraryhistory-client			5.0.0
@cdf/auth-devicecert				5.0.0
@cdf/auth-jwt					3.0.0
@cdf/bulkcerts 					6.0.0
@cdf/bulkcerts-client				2.0.0
@cdf/certificate-activator 			4.0.0
@cdf/certificate-vendor				5.0.0
@cdf/commands 					5.0.0
@cdf/commands-client				5.0.0
@cdf/config-inject				4.0.0
@cdf/deployment-helper				3.0.0
@cdf/device-monitoring				5.0.0
@cdf/device-patcher 				0.1.1 NEW
@cdf/device-patcher-client			1.1.1 NEW
@cdf/device-simulator-base			2.0.0
@cdf/errors					4.0.0
@cdf/events-alerts				3.0.0
@cdf/events-processor				4.0.0
@cdf/event-publisher				1.0.0 NEW
@cdf/express-middleware				3.0.0
@cdf/greengrass-deployment-client		REMOVED
@cdf/greengrass-provisioning-client 		REMOVED
@cdf/greengrass2-installer-config-generators	0.1.1 NEW
@cdf/greengrass2-provisioning 			0.1.1 NEW
@cdf/greengrass2-provisioning-client 		1.0.1 NEW
@cdf/installer 					0.1.2 NEW
@cdf/integration-tests				3.0.1
@cdf/lambda-invoke				3.0.0
@cdf/logger					4.0.0
@cdf/notifications-client			4.0.0
@cdf/provisioning 				5.0.0
@cdf/provisioning-client 			5.0.0
@cdf/simulation-launcher			3.0.0
@cdf/simulation-manager				3.0.0
@cdf/thing-list-builder				1.0.0 NEW
```

> The highlight of this release includes:
> - Support for Greengrass V2 provisioning (see [readme](https://github.com/aws/aws-connected-device-framework/tree/main/source/packages/services/greengrass2-provisioning))
> - Support for the remote installation, configuration, and updating of physical devices (see [readme](https://github.com/aws/aws-connected-device-framework/tree/main/source/packages/services/device-patcher))
> - A new cli wizard based installer that replaces the previous scripting approach (see [readme](https://github.com/aws/aws-connected-device-framework/tree/main/source/packages/services/installer))
> - Changes to how configuration is handled to allow us to perform tree shaking of the generated bundles in a future release
> 
> NOTE: BREAKING CHANGES: The changes to the installer and configuration hanlding are backward incompatible changes to the deployment process. Refer to [migration](https://github.com/aws/aws-connected-device-framework/blob/main/source/docs/migration.md) for details of how to migrate between releases

# Change Log - @cdf/assetlibrary

## 6.0.1
Thu, 17 Feb 2022 10:54:42 GMT

### Patches

- rollback the previous change where add associated roles to neptune

## 6.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

## 5.4.3
Fri, 04 Feb 2022 17:03:38 GMT

### Patches

- Fix bulk data import into Neptune by associating NeptuneLoadFromS3Role with Neptune cluster

## 5.4.2
Tue, 25 Jan 2022 19:57:26 GMT

# Change Log - @cdf/assetlibrary-client

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/assetlibrary-export

## 3.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/assetlibrary-history

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

## 4.1.4
Tue, 25 Jan 2022 19:57:26 GMT

### Patches

- Fixes an error-level log message in request handler to correctly show a status code

# Change Log - @cdf/assetlibraryhistory-client

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/auth-devicecert

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/bulkcerts

## 6.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

## 5.1.5
Tue, 25 Jan 2022 19:57:26 GMT

### Patches

- Fixes an error-level log message in request handler to correctly show a status code

# Change Log - @cdf/bulkcerts-client

## 2.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/certificate-activator

## 4.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/certificate-vendor

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/commands

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

## 4.2.4
Tue, 25 Jan 2022 19:57:26 GMT

### Patches

- Fixes an error-level log message in request handler to correctly show a status code

# Change Log - @cdf/commands-client

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/config-inject

## 4.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/deployment-helper

## 3.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/device-monitoring

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/device-patcher

## 0.1.1
Thu, 17 Feb 2022 09:27:38 GMT

_Initial release_

# Change Log - @cdf/device-patcher-client

## 1.1.1
Thu, 17 Feb 2022 09:27:38 GMT

_Initial release_

# Change Log - @cdf/device-simulator-base

## 2.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/errors

## 4.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/events-alerts

## 3.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/events-processor

## 4.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/event-publisher

## 1.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/express-middleware

## 3.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/greengrass2-installer-config-generators

## 0.1.1
Thu, 17 Feb 2022 09:27:38 GMT

_Initial release_

# Change Log - @cdf/greengrass2-provisioning

## 0.1.1
Thu, 17 Feb 2022 09:27:38 GMT

_Initial release_

# Change Log - @cdf/greengrass2-provisioning-client

## 1.0.1
Thu, 17 Feb 2022 09:27:38 GMT

_Initial release_

# Change Log - @cdf/installer

## 0.1.2
Thu, 17 Feb 2022 11:16:43 GMT

### Patches

- fix the filename for cfn-assetlibrary-export.yaml

## 0.1.1
Thu, 17 Feb 2022 09:27:38 GMT

_Initial release_

# Change Log - @cdf/integration-tests

## 3.0.1
Thu, 17 Feb 2022 12:56:53 GMT

### Patches

- increase the timeout before checking result

## 3.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/lambda-invoke

## 3.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/logger

## 4.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/notifications-client

## 4.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/provisioning

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/provisioning-client

## 5.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/simulation-launcher

## 3.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

# Change Log - @cdf/simulation-manager

## 3.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

## 2.0.4
Tue, 25 Jan 2022 19:57:26 GMT

### Patches

- Fixes an error-level log message in request handler to correctly show a status code

# Change Log - @cdf/thing-list-builder

## 1.0.0
Thu, 17 Feb 2022 09:27:38 GMT

### Breaking changes

- migrate configuration to use dotenv-flow and deployment to use installer

<hr /><em>This discussion was created from the release <a href='https://github.com/aws/aws-connected-device-framework/releases/tag/1.0.6'>1.0.6</a>.</em>

**Comment by aaronatbissell** [comment]
Whoa! Lots of changes! Excited to dig into this
